src/components/Global/ScreenOrientationLocker.tsx

```
@@ -0,0 +1,39 @@
+
+        lockOrientation()
+
+        const handleOrientationChange = () => {
+            // if the orientation is no longer portrait, try to lock it back.
+            if (screen.orientation && !screen.orientation.type.startsWith('portrait')) {
+                lockOrientation()
+            }
+        }
+
+        // some browsers might not support addEventListener on screen.orientation
+        if (screen.orientation && screen.orientation.addEventListener) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)